### PR TITLE
Add imx8mnevk and other imx fixes

### PR DESCRIPTION
**Pull request by vanmaegima**
## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for d76a50ca4b83f7c7b20e722bf646493b5e8633a8 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1675/docs/artifacts/html/index.html

File: source/reference-manual/boards/imx8mn.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Set up the board for updating using the manufacturing tools:
+
+.. figure:: /_static/boards/imx8mn-ddr4-evk.png
+     :width: 600
+     :align: center
+
+     imx8mn-ddr4-evk
+
+#. **OPTIONAL** - Only required if you have problems and/or want to see the boot console output.
```
Comment (kprosise) [inline source/reference-manual/boards/imx8mn.rst:21]:
```suggestion
#. **OPTIONAL** - Only required if you have problems or want to see the boot console output.
```
In most uses of "and/or" we can just use "or". This is a totally optional suggestion, feel free to keep as is if you would rather.

**Review by kprosise — APPROVED** [review]
One optional suggestion. LGTM

File: source/reference-manual/boards/imx8mn.rst (hunk 1 above)
Comment (vanmaegima) [reply source/reference-manual/boards/imx8mn.rst:21]: @kprosise I will keep it as it is right now as all of imx8m* flashing pages have this same text, but I will keep this in mind if we ever move that to a common page.

**Pull request merged**